datascience/tables.py

```diff
@@ -1202,23 +1246,21 @@ def sample(self, k=None, with_replacement=True, weights=None):
         """Returns a new table where k rows are randomly sampled from the
         original table.
 
-        Kwargs:
-            k (int or None): If None (default), all the rows in the table are
-                sampled. If an integer, k rows from the original table are
-                sampled.
+        Args:
+            ``k`` -- specifies the number of rows (``int``)  to be sampled from
+                self. Default is k is equal to number of rows in self.
 
-            with_replacement (bool): If True (default), samples the rows with
-                replacement. If False, samples the rows without replacement.
+            ``with_replacement`` -- (``boolean``), if true samples ``k`` rows
```

Comment:
nits: Technically there's only `bool`s in Python, not `boolean`s. Also, could you capitalize `true` and specify which option is the default? Finally, the same comment about using `self` applies here.